data/mods/PKs_Rebalance/pk_critter_fungus.json

```diff
@@ -0,0 +1,938 @@
+    "armor_stab":5,
+    "vision_day":35,
+    "vision_night":3,
+    "death_drops": {
+        "subtype": "collection",
+        "groups": [
+            ["mon_zombie_survivor_death_drops", 30],
+            ["default_zombie_death_drops", 60 ]
+        ]
+    },
+    "hp":105,
+    "death_function":["NORMAL", "FUNGUS"],
+    "special_attacks":[
+        ["FUNGUS", 180], ["SHRIEK", 120],
+        {
+            "type" : "bite",
+            "cooldown" : 20,
+            "min_mul" : 1.0
+        },
+        ["GRAB", 15]],
+    "description":"An apparent survivor of the apocolypse, this individual has visibly active fungal growth sprouting thoughout their body.  Whatever individuality they once held is completely subverted.",
+    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "STUMBLES", "GROUP_BASH", "POISON", "HUMAN", "LEATHER", "FAT", "BONES", "PUSH_MON"],
+    "//" : "There's a nerf of survivor gear coming around, and Ill be interested in seeing how it plays out."
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_furvivor_glock",
+    "name": "infected survivor",
+    "species":["MAMMAL", "FUNGUS"],
+    "default_faction":"finfected",
+    "symbol":"@",
+    "color":"ltred",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":15,
+    "aggression":3,
+    "morale":60,
+    "speed":100,
+    "melee_skill":6,
+    "melee_dice":2,
+    "melee_dice_sides":6,
+    "melee_cut":8,
+    "dodge":4,
+    "armor_bash":7,
+    "armor_cut":7,
+    "armor_stab":3,
+    "armor_acid":1,
+    "vision_day":50,
+    "vision_night":2,
+    "death_drops": {
+        "subtype": "collection",
+        "groups": [
+            [ "furvivor_glock", 100 ]
+        ]
+    },
+    "hp":135,
+    "death_function":["NORMAL", "FUNGUS"],
+    "special_attacks":[
+        ["PARROT", 40], ["FUNGUS", 300],
+        {
+            "type" : "bite",
+            "cooldown" : 30,
+            "min_mul" : 1.0
+        },
+        ["GRAB", 12],
+                {
+            "type" : "gun",
+            "cooldown" : 2,
+            "gun_type" : "glock_19",
+            "ammo_type" : "9mm",
+            "max_ammo" : 1000,
+            "fake_str" : 11,
+            "fake_dex" : 7,
+            "fake_per" : 5,
+            "fake_skills" : [["gun", 3], ["pistol", 3]],
+            "move_cost" : 105,
+            "targeting_cost" : 50,
+            "range" : 13,
+            "description" : "The infected survivor fires wildly!",
+            "targeting_sound" : "incoherent gibberish!",
+            "targeting_volume" : 15,
+            "no_ammo_sound" : "a chk!"
+        }],
+    "starting_ammo" : {"9mm": 31},
+    "description":"A survivor of the apocolypse, this individual is succumbbing to the mycus, paranoia, and incoherency.  Unpredicable and terminally-ill, they will not respond to any attempts to communicate.  You can't help but notice they still wear their armor and holster, despite the fresh fungal budding that threatens to rip through it  . . .",
```

Comment:
Also, replacing of "  . . ." by a proper ellipsis symbol (…) could be probably better.